Scuti/Scripts/UI/Utilities/GamepadCursor.cs

```diff
@@ -0,0 +1,162 @@
+﻿using System.Collections;
```

Comment:
@Jpablogs8 I don't think we want to include our own cursor. The games that integrate our SDK will already have their own cursor. (I think).

Reply:
Sorry Mark, what do you mean here, the image, or do you mean the code that drives a cursor with the gamepad joystick? Or something else?

Reply:
I think all of it but I am not certain.

Lets say a game developer has a game that works with a controller. They should already have a cursor and controller setup. So adding our own would create 2 cursors on the screen. 

The task for the PR is to add controller support to scuti so that a game that already has a controller can use scuti. You'd want to create a game that has controller support and then embed Scuti into it and make sure it works. So maybe the input code and this GamepadCursor should be added to a branch in Scuti-App and tested there, but I don't think this code should be in Scuti-sdk.

Reply:
Ahh yes, I get your point! You're right! Okay then I'll remove the script from this PR and make a new PR for scuti-app, do you think?

Reply:
Sounds perfect